src/cpu/x64/jit_generator.hpp

```diff
@@ -721,7 +721,8 @@ class jit_generator : public Xbyak::CodeGenerator, public c_compatible {
             // previously there was "subps(x, op2)" for some reason
             vsubss(x, op1, op2);
         else {
-            assert(x.isEqualIfNotInherited(op1));
+            if (!x.isEqualIfNotInherited(op1))
+                movss(x, op1);
```

Comment:
Could you add braces for better readability and maintainability ?

Reply:
Any codestyle regulation on this or it's your own preference?

Reply:
Just an observation that such code is mostly error-prone, because indents after if statement does not mean anything in C++ and it could confuse. In contrast, the curly braces is hard to misinterpret code

Reply:
I don't see necessity to change this since the syntax is correct and there is no code style regulation about this.